Bottom: aecfe146bb6a8c85fc043fb6406b9991777537bc
Top:    e9d1fa5b746f5d40ed2e08db7be1d6ebdaa6c711
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-04 15:39:46 +0000

crosshair.c: When drawing via DRC clearence, pass correct angle.

360 * 64 was a hold-over from before the HID split. The passed
angle should be 360.



---

diff --git a/src/crosshair.c b/src/crosshair.c
index 42fd81c..5335cd1 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -593,7 +593,7 @@ DrawAttached (Boolean BlockToo)
 	  s = Settings.ViaThickness / 2 + PCB->Bloat + 1;
 	  gui->set_color (Crosshair.GC, Settings.CrossColor);
 	  gui->draw_arc (Crosshair.GC,
-			 Crosshair.X, Crosshair.Y, s, s, 0, 360 * 64);
+			 Crosshair.X, Crosshair.Y, s, s, 0, 360);
 	  gui->set_color (Crosshair.GC, Settings.CrosshairColor);
 	}
       break;
